main.go

```diff
@@ -198,33 +206,54 @@ func initFlags(fs *pflag.FlagSet) {
 		fmt.Sprintf("Label value that the controller watches to reconcile cluster-api objects. Label key is always %s. If unspecified, the controller watches for all cluster-api objects.", clusterv1.WatchLabel),
 	)
 
-	flag.DurationVar(
+	fs.DurationVar(
 		&syncPeriod,
 		"sync-period",
 		10*time.Minute,
 		"The minimum interval at which watched resources are reconciled (e.g. 15m)",
 	)
 
-	flag.IntVar(
+	fs.IntVar(
 		&webhookPort,
 		"webhook-port",
 		9443,
 		"Webhook Server port",
 	)
 
-	flag.StringVar(
+	fs.StringVar(
 		&webhookCertDir,
 		"webhook-cert-dir",
 		"/tmp/k8s-webhook-server/serving-certs/",
 		"Webhook cert dir, only used when webhook-port is specified.",
 	)
 
-	flag.StringVar(
+	fs.StringVar(
 		&healthAddr,
 		"health-addr",
 		":9440",
 		"The address the health endpoint binds to.",
 	)
+
+	fs.IntVar(&metal3MachineConcurrency, "metal3Machine-concurrency", 10,
```

Comment:
In CAPI they seem to [stick to all lower case](https://github.com/kubernetes-sigs/cluster-api/blob/838dcbbb20c45fc5966727151f9ca1320dfcd783/main.go#L163) for the flags, maybe we should do the same? (Same for the rest of the concurrency flags)
```suggestion
	fs.IntVar(&metal3MachineConcurrency, "metal3machine-concurrency", 10,
```

Reply:
yes I thought about it and later choose to just go this way. I can change it.

Reply:
I have no strong preference on this but I think it is quite common to keep flags lower case so maybe it is better that way

Reply:
Please do not put upper case characters into flags, I can guarantee nobody can write correctly if you do. :)

Reply:
Done